webpack.config.js

```diff
@@ -41,6 +41,11 @@ const shouldCheckForCycles = process.env.CHECK_CYCLES === 'true';
 const codeSplit = config.isEnabled( 'code-splitting' );
 const isCalypsoClient = process.env.CALYPSO_CLIENT === 'true';
 
+const workerCount =
```

Comment:
we can split this calc off to a utility function and require it both places

Reply:
Sure, the entire module could be:

```js
module.exports = process.env.CIRCLECI
	? 2
	: process.env.WORKERS && ! Number.isNaN( parseInt( process.env.WORKERS, 10 ) )
		? Math.max( 1, parseInt( process.env.WORKERS, 10 ) )
		: Math.max( 2, Math.floor( require( 'os' ).cpus().length / 2 ) );
```

I just don't know where that should go. There's nothing like that around Calypso right now.